interface/ceed-basis.c

```diff
@@ -1739,9 +1739,9 @@ CeedPragmaOptimizeOn
 
     /**
       @brief Return Simultaneous Diagonalization of two matrices.
-               This solves the generalized eigenvalue problem A x = lambda B x, where A and B are symmetric and B is positive definite.
-               We generate the matrix X and vector Lambda such that X^T A X = Lambda and X^T B X = I.
-               This is equivalent to the LAPACK routine 'sygv' with TYPE = 1.
+               This solves the generalized eigenvalue problem \f$A x = \lambda B x\f$, where \f$A\f$ and \f$B\f$ are symmetric and \f$B\f$ is positive
+    definite. We generate the matrix \f$X\f$ and diagonal \f$\Lambda\f$ such that \f$X^T A X = \Lambda\f$ and \f$X^T B X = I\f$. This is equivalent to
+    the LAPACK routine 'sygv' with `TYPE = 1`.
```

Comment:
Note - I try to use one sentence per line with the documentation so we can change one sentence and have it only effect that one line. The `clang-tidy` PR did break some of the docstrings wrt this convention though and I haven't had a chance to go fix them all.

Reply:
Yeah, that makes sense. I usually do that for my markdown/LaTeX files for the same reason.

Reply:
Speaking of the developer notes, I notice that include-what-you-use doesn't seem to be used. At least it complains about a lot of core files not having the correct includes. I know fluids is quite guilty of it too.

Reply:
Its been a while since I ran IWYU. It produces some silly results sometimes, so its something I do manually.